packages/blockchain/src/index.ts

```diff
@@ -20,6 +19,21 @@ const Stoplight = require('flow-stoplight')
 const level = require('level-mem')
 const semaphore = require('semaphore')
 
+// promisify a function: resolves this promise if callback is called
```

Comment:
nice, you can actually just `import { promisify } from 'util'`

Reply:
This is an external package right? I am not sure if this does what this intends to do; this is a much simpler variant I think.

Reply:
Ah, no, this is a build-in Node.js functionality (see link from Ryan) and we are [using this all over the place](https://github.com/search?q=org%3Aethereumjs+util.promisify&type=Code). Please use here as well, otherwise this bloats the code unnecessarily.

Reply:
I was not aware of it, I thought it was an external package. Will change, thanks!

Reply:
great :) if it has trouble erroring out in karma test runner then you might need to import from `util-promisify` for a browser-friendly version, as used elsewhere in the monorepo, but it's a very small package and we can hopefully remove the need for it soon.